schema/tables/disk_info.yml

```diff
@@ -0,0 +1,26 @@
+name: disk_info
+platforms:
+  - chrome
+description: Returns information about disks found on the system. A separate line for each disk found.
+examples: >-
+  ```
+  SELECT * FROM disk_info;
+  ```
+columns:
+  - name: capacity
+    type: integer
+    description: Capacity on the disk in Bytes.
+    required: false
+  - name: id
+    type: string
+    description: UUID of the disk.
+    required: false
+  - name: name
+    type: string
+    description: Name of the disk.
+    required: false
+  - name: type
+    type: string
+    description: Type of the disk (e.g. fixed).
+    required: false
```

Comment:
```suggestion
columns:
  - name: capacity
    type: integer
    description: Capacity on the disk in Bytes.
    required: false
    platforms:
       - chrome
```

Reply:
@eashaw In parallel to you writing this comment I merged all the new fields and called them the same name as existing fields.

Please review.

Reply:
@sharon-fdm Sorry for the confusion. For this table override file, because only four columns are supported on ChromeOS, we will need to add overrides for the columns that **are only** supported on Windows.

For the columns that are not supported on ChromeOS, we only need to add the name of the column and the platform e.g.,

```
columns:
  - name: partitions
    platforms:
      - windows
```